tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>2.32.0</Version>
```

Comment:
I'm worried that users might want to use instrumentation introduced in a newer version (e.g. 2.35.0) but they published their app with an older version of the trimming package (e.g. 2.32.0). While we cannot control this, can we document either in the NuGet package or in our internal docs how we will detect/handle this scenario when it comes up?

Reply:
Do we need to keep this version in sync with the tracer? Are we planning to publish a new `Datadog.Trace.Trimming` package on every release? Or can it stay mostly static except when there's a change?

Reply:
> Are we planning to publish a new Datadog.Trace.Trimming package on every release?

Yes, we will need to, because we will need to update the trimming results. 

I wonder if Datadog.Trace should transitively reference Datadog.Trace.Trimming too 🤔 

Also note that in a version conflict scenario, all bets are off here 😬  We should probably make that _Very_ explicit in the package readme?